pkgs/by-name/re/redis/package.nix

```
@@ -100,15 +104,25 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postCheck
   '';
 
-  passthru.tests.redis = nixosTests.redis;
-  passthru.serverBin = "redis-server";
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgram = "${placeholder "out"}/bin/${finalAttrs.pname}-server";
```

```suggestion
  versionCheckProgram = "${placeholder "out"}/bin/redis-server";
```
see the pname tracking issue